protocolize-api/src/main/java/dev/simplix/protocolize/api/item/Attribute.java

```diff
@@ -0,0 +1,43 @@
+package dev.simplix.protocolize.api.item;
+
+import dev.simplix.protocolize.data.AttributeType;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Attribute {
+
+    AttributeType type;
```

Comment:
Is there a reason why those fields are not private?

Reply:
fixed